### DISCUSSION TITLE
Error after recent updates: "Error while downloading blink.cmp pre-built binary"

### DISCUSSION THREAD

**Discussion by SargisPlusPlus** [LazyVim/LazyVim#4793] [Q&A] [answered]
Full error:
```
Error while downloading blink.cmp pre-built binary: Can't download from github due to not being on a git tag and no fuzzy.prebuilt_binaries.force_version set, but found no built version of the library. Either run `cargo build --release` via your package manager, switch to a git tag, or set `fuzzy.prebuilt_binaries.force_version` in config. See the README for more info.
```

Happens on start everytime. 

My config:
```
{
  "extras": [
    "lazyvim.plugins.extras.ai.copilot",
    "lazyvim.plugins.extras.ai.copilot-chat",
    "lazyvim.plugins.extras.coding.mini-surround",
    "lazyvim.plugins.extras.formatting.prettier",
    "lazyvim.plugins.extras.lang.json",
    "lazyvim.plugins.extras.lang.tailwind"
  ],
  "news": {
    "NEWS.md": "7107"
  },
  "version": 7
}

```

OS: MacOs 14.6
NVIM v0.10.0

**Comment by dpetka2001** [comment]
You don't have the `blink` Extra enabled, so you probably have some kind of misconfiguration in your personal files.

**Comment by folke** [comment]
You need to enable the blink extra to have a proper setup.
Blink has a binary component, so you need to follow releases with `version = "*"`.
You can also enable the `main` branch in the blink extra. See the blink extra docs.

**Reply by SargisPlusPlus** [reply]
I do have `blink.cmp` in my lazy-lock.json
```
{
...
  "blink-cmp-copilot": { "branch": "main", "commit": "e3eedf36ceda1932e296ec894de4ceea8ddeacb6" },
  "blink.cmp": { "branch": "main", "commit": "1309ec5babfc897da7927ffabe7d4bea5bb9b0cc" },
...
}
```

But thank you for response. I will try to figure out what update they pushed out that caused it break. I didn't install any new packages aside from updating my current setup. so it has to be related to just `blink` bad update.

**Reply by dpetka2001** [reply, marked as answer]
You don't have it enabled via `:LazyExtras`. Open `:LazyExtras`, find `coding.blink` and press `x` to enable it. Then restart Neovim for the plugin spec to take effect and it will download the `blink` binary.

**Reply by SargisPlusPlus** [reply]
@dpetka2001 ah i see. thank you! this did it!

**Comment by folke** [comment]
fyi, there was indeed a bug in the copilot spec that always loaded blink. See https://github.com/LazyVim/LazyVim/issues/4795

Should be fixed now.
